packages/react/src/utils/hooks/useManagedRef.ts

```
@@ -3,7 +3,7 @@ import { useCallback, useRef } from "react";
 export const useManagedRef = <TNode>(
   callback: (node: TNode) => (() => void) | void,
 ) => {
-  const cleanupRef = useRef<(() => void) | void>();
+  const cleanupRef = useRef<(() => void) | void>(undefined);
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)